pkg/action/yaml.def.go

```diff
@@ -284,9 +285,16 @@ type Option struct {
 	Type        jsonschema.Type `yaml:"type"`
 	Default     interface{}     `yaml:"default"`
 	Required    bool            `yaml:"required"` // @todo that conflicts with json schema object definition
+	Process     []Process       `yaml:"process"`
 	RawMap      map[string]interface{}
 }
 
+// Process stores information about processor and options that should be applied to processor.
+type Process struct {
```

Comment:
Maybe it's better to rename to `ValueProcessDef` since it's a plain struct and used in definition only.

Reply:
done